Sources/Session.php

```diff
@@ -29,9 +29,7 @@ function loadSession()
 
 	// Attempt to change a few PHP settings.
 	@ini_set('session.use_cookies', true);
-	@ini_set('session.use_only_cookies', false);
```

Comment:
Even though this defaults to true, it would made sense to just set it true instead of removing it.

Reply:
A compromise may be to add this in a future PR for the installer/upgrader.  If it finds the variable set, it warns about it.

Reply:
I know that php default is true, I just wanted to keep it for safeguarding bad server configurations. (like manually set to false on php.ini or httpd.conf)

But yeah, on properly configured php installations, removing it has the same effect as setting it to true and it looks cleaner.